docs/docs/pipeline/authn.md

```diff
@@ -328,6 +328,129 @@ HTTP/1.0 401 Status Unauthorized
 The request is not authorized because the provided credentials are invalid.
 ```
 
+## `bearer_token`
+
+The `bearer_token` authenticator will forward the request method, path and
+headers to a session store. If the session store returns `200 OK` and body
+`{ "subject": "...", "extra": {} }` then the authenticator will set the subject
+appropriately.
+
+### Configuration
+
+- `check_session_url` (string, required) - The session store to forward request
+  method/path/headers to for validation.
+- `preserve_path` (boolean, optional) - If set, any path in `check_session_url`
+  will be preserved instead of replacing the path with the path of the request
+  being checked.
+- `extra_from` (string, optional - defaults to `extra`) - A
+  [GJSON Path](https://github.com/tidwall/gjson/blob/master/SYNTAX.md) pointing
+  to the `extra` field. This defaults to `extra`, but it could also be `@this`
+  (for the root element), `session.foo.bar` for
+  `{ "subject": "...", "session": { "foo": {"bar": "whatever"} } }`, and so on.
+- `subject_from` (string, optional - defaults to `subject`) - A
+  [GJSON Path](https://github.com/tidwall/gjson/blob/master/SYNTAX.md) pointing
+  to the `subject` field. This defaults to `subject`. Example: `identity.id` for
```

Comment:
```suggestion
  to the `subject` field. This defaults to `sub`. Example: `identity.id` for
```

Reply:
I've just sent in the change. This does make it diverge from the cookie_session configuration, but perhaps that will get the change sometime in the future as well?